Samples/3a-literate-swagger/shell/stderr.txt

```diff
@@ -15,6 +15,8 @@ ERROR (XmsExamplesRequired/R2022/SDKViolation): Please provide x-ms-examples des
     - /Samples/3a-literate-swagger/swagger.md:178:4 ($.paths["/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Search/searchServices/{serviceName}"].delete)
 ERROR (XmsExamplesRequired/R2022/SDKViolation): Please provide x-ms-examples describing minimum/maximum property set for response/request payloads for operations. Operation: 'Services_List'
     - /Samples/3a-literate-swagger/swagger.md:207:4 ($.paths["/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Search/searchServices"].get)
+WARNING (LongRunningOperationsWithLongRunningExtension/R2065/SDKViolation): A long running operation must have 'x-ms-long-running-operation' extension enabled.
```

Comment:
is this /subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Search/searchServices/{serviceName} PUT a long running operation for https://github.com/Azure/autorest/blob/master/Samples/3a-literate-swagger/swagger.md?